README.md

```diff
@@ -71,9 +68,9 @@ case class CaseClassA(
   fieldGpb: CaseClassB,
   fieldGpbOption: Option[CaseClassB],
   fieldGpbOptionEmpty: Option[CaseClassB],
-  fieldStringsList: immutable.Seq[String],
-  fieldOptionIntegersList: Option[List[Int]],
-  fieldOptionIntegersEmptyList: Option[List[Int]]
+  fieldStrings: immutable.Seq[String],
+  fieldOptionIntegers: Option[List[Int]],
```

Comment:
Do you really think it would be nice to use like this? I would probably understand if it meant that the underlying GPB had the list actually set (hasList returns true) but otherwise I would rather have an empty list as an indication of "nothing".